aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -1089,6 +1090,7 @@ Tensor& rand_out(
 
 Tensor rand_like(
     const Tensor& self,
+    std::optional<Generator> generator,
```

Comment:
It is better to append the new parameter `generator` to the parameter list.

Reply:
If is the html doc, then I already done it. Please check [built html preview](https://docs-preview.pytorch.org/pytorch/pytorch/166160/generated/torch.rand_like.html) out.

Reply:
> Sorry. A little bit confused. To which parameter list?

rand_like gets a new parameter `generator`, it is better to put it at the end.

Reply:
A question, if we move `generator` to the end of the list. Should we also change the corresponding python function parameter list? Which will break the consistency to funcitons without `_like` and I think is not worth to do that. 🤔

Reply:
I will try it locally.